README.md

```diff
@@ -24,102 +25,80 @@
 | cw20-ics20         | [Release v0.13.4](https://github.com/CosmWasm/cw-plus/releases/download/v0.13.4/cw20_ics20.wasm)         | [![Docs](https://docs.rs/cw20-ics20/badge.svg)](https://docs.rs/cw20-ics20)                 | [![codecov](https://codecov.io/gh/CosmWasm/cw-plus/branch/main/graph/badge.svg?token=IYY72ZVS3X)](https://codecov.io/gh/CosmWasm/cw-plus) |
 
-Note: `cw721` and `cw721-base` have moved to the new [`cw-nfts` repo](https://github.com/CosmWasm/cw-nfts)
-and can be followed there.
+Note: `cw721` and `cw721-base` have moved to the new [`cw-nfts` repo](https://github.com/CosmWasm/cw-nfts) and can be
+followed there.
 
-Note: most of the `cw20-*` contracts besides `cw20-base` have moved to the new [`cw-tokens` repo](https://github.com/CosmWasm/cw-tokens)
-and can be followed there.
+Note: most of the `cw20-*` contracts besides `cw20-base` have moved to the new
+[`cw-tokens` repo](https://github.com/CosmWasm/cw-tokens) and can be followed there.
 
-This is a collection of specification and contracts designed for
-use on real networks. They are designed not just as examples, but to
-solve real-world use cases, and to provide a reusable basis to build
-many custom contracts.
+This is a collection of specification and contracts designed for use on real networks. They are designed not just as
+examples, but to solve real-world use cases, and to provide a reusable basis to build many custom contracts.
 
-If you don't know what CosmWasm is, please check out
-[our homepage](https://cosmwasm.com) and
-[our documentation](https://docs.cosmwasm.com) to get more background.
-We are running [public testnets](https://github.com/CosmWasm/testnets#running)
-you can use to test out any contracts.
+If you don't know what CosmWasm is, please check out [our homepage](https://cosmwasm.com) and
+[our documentation](https://docs.cosmwasm.com) to get more background. We are running
+[public testnets](https://github.com/CosmWasm/testnets#running) you can use to test out any contracts.
 
-**Warning** None of these contracts have been audited and no liability is
-assumed for the use of this code. They are provided to turbo-start
-your projects.
+**Warning** None of these contracts have been audited and no liability is assumed for the use of this code. They are
+provided to turbo-start your projects.
 
-**Note** All code in pre-1.0 packages is in "draft" form, meaning it may
-undergo minor changes and additions until 1.0. For example between 0.1 and
-0.2 we adjusted the `Expiration` type to make the JSON representation
-cleaner (before: `expires: {at_height: {height: 12345}}` after
-`expires: {at_height: 12345}`)
+**Note** All code in pre-1.0 packages is in "draft" form, meaning it may undergo minor changes and additions until 1.0.
```

Comment:
Do we want to remove this, as we plan to go to 1.0? At least that reference to very old changes

Reply:
Yeah, let's get rid of that actually